contracts/MyMultiSig.cdc

```diff
@@ -57,74 +66,60 @@ pub contract MyMultiSig {
             let keyList = Crypto.KeyList()
             let account = getAccount(acctAddress)
 
-            let publicKeys: [[UInt8]] = []
-            let weights: [UFix64] = []
-            let signAlgos: [UInt] = []
+            
             
             let uniqueKeys: {Int: Bool} = {}
             for id in keyIds {
                 uniqueKeys[id] = true
```

Comment:
Maybe we can move the assertion that checks the lengths here to a fail fast approach, for example
```
for id in keyIds {
      assert(uniqueKeys[id] != nil, message: "Invalid duplicates of the same keyID provided for signature")
      uniqueKeys[id] = true
}
```

Reply:
oops, now i see that we use signatures[i] too, so might need to ignore this one 😄